app/scripts/utils/routes.ts

```diff
@@ -1,11 +1,24 @@
 import vedaThematics, {
   DatasetData,
   VedaDatum,
-  ThematicData
+  ThematicData,
+  DiscoveryData
 } from 'veda/thematics';
 
 type ThematicOrString = VedaDatum<ThematicData> | string;
-type DatasetOrString = VedaDatum<DatasetData> | string;
+
+export const ABOUT_PATH = '/about';
+export const DISCOVERIES_PATH = '/discoveries';
+export const DATASETS_PATH = '/data-catalog';
```

Comment:
I'm wondering if the path for the datasets shouldn't be `/datasets/:id`. thoughts?